### PR TITLE
Feature for mapping. setWorldSpecialPropertyEnabled, `underworldwarp` property

### PR DESCRIPTION
**Pull request by CrosRoad95**
New property `underworldwarp`, by default: true.
If `true` then you can't fall under z -100.
if `false` then you can fall up to -5000.
Currently you can't build examples caves under z coordinate -100, this could be useful for mappers, mainly for DayZ servers.

Example test resource which allow to fall under map up to z = -5000 for peds and cars
```
setWorldSpecialPropertyEnabled("underworldwarp", false)
```
effect:
under map

Thanks for Saml1er and Octalype.

### CODE REVIEW

**Comment by Pirulax** [comment]
In my option those commit messages aren't useful, next time please write some info about the commit into the commit message.

**Comment by qaisjp** [comment]
Is there a reason why you would need only one of these functions and not always both? If not, please can you merge this into one function (in both game_sa and the Lua defs)?

**Comment by CrosRoad95** [comment]
Yes i can marge this two functions, do you have any good name for her?

**Comment by ccw808** [comment]
Just an idea, but what about using https://wiki.multitheftauto.com/wiki/SetWorldSpecialPropertyEnabled

**Comment by qaisjp** [comment]
Yep, @CrosRoad95 I was going to move it into setWorldSpecialPropertyEnabled for you when merging, but you can do that instead, if you want!

I was going to go for `PROP_UNDERWORLD_WARP` and use the string "underworldwarp". Not sure about the name though.

**Comment by CrosRoad95** [comment]
i will do it, only give me name of this property
today will be update, rewrited to work in `setWorldSpecialPropertyEnabled` function

**Review by qaisjp — APPROVED** [review]
LGTM outside `game_sa` changes. Someone please merge if the `game_sa` changes are fine.

File: Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.h [hunk 1]
```diff
@@ -176,6 +176,10 @@ class CLuaFunctionDefs
     LUA_DECLARE(SetFPSLimit);
     LUA_DECLARE(GetFPSLimit);
     LUA_DECLARE(FetchRemote);
+    LUA_DECLARE(SetFallenPedsEnabled);
+    LUA_DECLARE(SetFallenCarsEnabled);
+    LUA_DECLARE(GetFallenPedsEnabled);
+    LUA_DECLARE(GetFallenCarsEnabled);
```
Comment (ccw808) [inline Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.h]: Are these still needed?

File: Client/game_sa/CGameSA.h [hunk 2]
```diff
@@ -504,7 +509,7 @@ class CGameSA : public CGame
     static float*         VAR_OldTimeStep;
     static float*         VAR_TimeStep;
     static unsigned long* VAR_Framelimiter;
-
+
```
Comment (ccw808) [inline Client/game_sa/CGameSA.h]: White space inconstancy

**Pull request merged**